pyredner/render_pytorch.py

```diff
@@ -583,13 +615,19 @@ def backward(ctx,
             d_diffuse_list.append(d_diffuse)
             d_specular_list.append(d_specular)
             d_roughness_list.append(d_roughness)
+            d_generic_list.append(d_generic)
             d_normal_map_list.append(d_normal_map)
             d_diffuse_uv_scale = torch.zeros(2, device = pyredner.get_device())
             d_specular_uv_scale = torch.zeros(2, device = pyredner.get_device())
             d_roughness_uv_scale = torch.zeros(2, device = pyredner.get_device())
+            d_generic_uv_scale = torch.zeros(2, device = pyredner.get_device())
```

Comment:
This line is unnecessary since L627 handles it already.

Reply:
ok done